docs/openapi/schemas.yaml

```diff
@@ -933,3 +933,84 @@ SiteMetricList:
   type: array
   items:
     $ref: './schemas.yaml#/SiteMetric'
+ImportOptions:
+  type: object
+  properties:
+    pageLoadTimeoutMs:
+      description: The time in milliseconds to wait for the page to load
+      type: integer
+    enableJavaScript:
+      description: Whether to enable JavaScript on the page
+      type: boolean
+    scrollToBottom:
+      description: Whether to scroll to the bottom of the page
+      type: boolean
+    customHeaders:
+      description: Custom headers to send with each HTTP request
+      type: object
+      additionalProperties: true
+  example:
+    pageLoadTimeoutMs: 100
+    enableJavaScript: true
+    scrollToBottom: true
+    customHeaders:
+      example-header-name: 'example-header-value'
+ImportJobRequest:
+  type: object
+  required:
+    - urls
+  properties:
+    urls:
+      description: The URLs to import
+      type: array
+      items:
+        $ref: '#/URL'
+    options:
+      description: Optional configuration params, defined as a JSON object
+      $ref: '#/ImportOptions'
+    importScript:
+      description: The import.js mapping file
```

Comment:
is it required? if so, add to `required` array; if not please mention `Optional` in the description as you did for `options`